src/RichTextEditor/FormattingToolbar/ToolbarMenu.js

```diff
@@ -25,4 +25,6 @@ const ToolbarMenu = React.forwardRef(
   ({ ...props }, ref) => (<Menu {...props} ref={ref} />)
 );
 
+ToolbarMenu.displayName = 'ToolbarMenu';
```

Comment:
Can you remove the `// eslint-disable-next-line react/display-name` from above here too?

Reply:
Yes, good catch!